### PR TITLE
Update ausserhalb-des-studiums.md

### PR DESCRIPTION
**Pull request by blackcatfighter**

### CODE REVIEW

**Comment by blackcatfighter** [comment]
Infos eingefügt

File: content/faq/ausserhalb-des-studiums.md [hunk 1]
```diff
@@ -5,16 +5,56 @@ header:
...
   image: '/assets/backgrounds/freizeit.jpg'
 ---
-### Frage 1
+Ihr seid zum Studieren nach Karlsruhe gezogen und wollt wissen, welche Freizeitaktivitäten hier möglich sind? 
+Am Anfang des Sommersemesters könnt ihr euch für 8€ das Studierenden-Quartett in der hochschuleigenen Cafeteria im A-Gebäude holen. Kostenloser Kaffee, reduzierte (nicht)-alkoholische Getränke und vergünstigte Freizeitaktivitäten warten auf euch! Das Quartett ist für ein Jahr gültig. 
+Zudem gibt es hier weitere Anregungen. Nach Vorlage eures gültigen Studierendenausweises gibt es dabei oft Vergünstigungen für Studierende. 
+### Schloss/Schlossgarten:
+Im Schloss ist das „Badische Landesmuseum“. Der dahinterliegende Schlossgarten ist ein beliebter Freizeit- und Erholungsort. 
+### Günther-Klotz-Anlage (= „Klotze“):
+Auch die Günther-Klotz-Anlage eignet sich durch die Grünanlage als Freizeit- und Erholungsort. Das bekannte „Fest“ findet jährlich im Sommer auf der „Klotze“ statt.
```
Comment (Wekra) [inline content/faq/ausserhalb-des-studiums.md:14]: "[...] auf der Klotze" or rather "[...] in der Klotze" ? 🤔

File: content/faq/ausserhalb-des-studiums.md [hunk 2]
```diff
@@ -5,16 +5,56 @@ header:
...
-### Frage 1
+Ihr seid zum Studieren nach Karlsruhe gezogen und wollt wissen, welche Freizeitaktivitäten hier möglich sind? 
+Am Anfang des Sommersemesters könnt ihr euch für 8€ das Studierenden-Quartett in der hochschuleigenen Cafeteria im A-Gebäude holen. Kostenloser Kaffee, reduzierte (nicht)-alkoholische Getränke und vergünstigte Freizeitaktivitäten warten auf euch! Das Quartett ist für ein Jahr gültig. 
+Zudem gibt es hier weitere Anregungen. Nach Vorlage eures gültigen Studierendenausweises gibt es dabei oft Vergünstigungen für Studierende. 
+### Schloss/Schlossgarten:
+Im Schloss ist das „Badische Landesmuseum“. Der dahinterliegende Schlossgarten ist ein beliebter Freizeit- und Erholungsort. 
+### Günther-Klotz-Anlage (= „Klotze“):
+Auch die Günther-Klotz-Anlage eignet sich durch die Grünanlage als Freizeit- und Erholungsort. Das bekannte „Fest“ findet jährlich im Sommer auf der „Klotze“ statt. 
+### Kinos:
+Filmpalast, Universum, Schauburg, Die Kurbel
```
Comment (Wekra) [inline content/faq/ausserhalb-des-studiums.md:16]: 'Die Kurbel' is [insolvent as of 2018](https://ka.stadtwiki.net/Die_Kurbel).

**Review by Wekra — CHANGES_REQUESTED** [review]
Add some general feedback regarding the content.

**Comment by Wekra** [comment]
Resolved in #16 and will therefore be closed.

**Comment by Wekra** [comment]
@blackcatfighter do you want to keep the patch-branch open?

**Pull request closed without merging**